apps/bajour/src/components/briefing/basel-briefing.tsx

```diff
@@ -391,9 +397,23 @@ export const BaselBriefing = ({teasers, blockStyle}: BaselBriefingProps) => {
                 </Author>
               )}
 
-              <ReadMoreButton variant="outlined" color="inherit" size="small">
-                Ganzes Briefing
-              </ReadMoreButton>
+              <ButtonRow>
+                {(blockStyle as BriefingType) === BriefingType.BaselBriefing && (
+                  <ReadMoreButton
```

Comment:
Should use LinkComponent as property instead of using onClick as this now breaks browser semantics.

Reply:
@Itrulia The entire component is already wrapped in a `<LinkWrapper>` so the entire element is clickable. Nesting interactive elements is not allowed. We could wrap the individual parts in a link though. What do you think?

Reply:
Then it shouldn't be interactable else we fail WCAG guidelines. 
Either have the elements themselves interactable or move the Spotify link out into a different part (and design)